### PR TITLE
[WIP] DJ centered alternative

### PR DESCRIPTION
**Pull request by dnk8n**
My plan is to maintain a more DJ centered version of the repository.

The idea is for DJs to easily download a single track, or playlist. Seldom are we interested in full albums, etc. I also aim to bring in more meta information which might be able to be filtered upon on the decks (or at least RekordBox / Engine / Serato).

Contributions welcome. Please feel free to comment.

Next plan is to also compile an m3u playlist along with the download so that a single file can be dragged in to the DJ software, and it links to the actual files.

**Disclaimer**

This tool is intended for temporary downloads. Explore the music, try different tracks on. Consider supporting artists by legitimately paying for your own Deezer subscription, and once you are earning money as a DJ, consider purchasing a copy directly from them.

In short, this tool is useful and intended for brainstorming, for private use.

### CODE REVIEW

**Comment by dnk8n** [comment]
Didn't actually mean to add the PR to the upstream. But I guess maybe this will potentially attract more collaborators. Feel free to close if you wish.

**Comment by dnk8n** [comment]
@sferaggio I missed your invitation to collaborate here. Can maybe discuss, or I just continue on my fork.
